phpstan-baseline.neon

```diff
@@ -20,6 +20,21 @@ parameters:
 			count: 2
 			path: src/Executor/Executor.php
 
+		-
+			message: "#^Switch condition type \\(true\\) does not match case condition \\$selection instanceof \\\\GraphQL\\\\Language\\\\AST\\\\FieldNode \\(false\\)\\.$#"
```

Comment:
I have no idea where the hell these errors came from; I didn't touch ReferenceExecutor...